test/controllers/concerns/authentication_test.rb

```diff
@@ -172,6 +176,35 @@ def raising
       assert_not User.current, 'current user must be reset after request'
     end
 
+    should 'account found, is_internal unset, updates it' do
```

Comment:
A slight variant, which I expect to hit in prod: 'account found, is_internal set, updates it'

Reply:
Do we really have that as a use-case in practice? If the `is_internal` is set, it means that we already know if the account is internal or not and that AFAIK can't change in the future :thinking:

Reply:
Did you run a query on accounts in prod to see what the state of this existing column is? How are you sure they are set properly there?

Reply:
Yes, they're all set to `nil`